src/models/network.ts

```diff
@@ -28,32 +28,35 @@ const versionBytes = {
 // pubkeyhash: prefix for p2pkh addresses
 // scripthash: prefix for p2sh addresses
 // privatekey: prefix for private key WIF (Wallet Import Format)
+// bech32prefix: prefix for bech32 addresses (we will use 'bc' for both mainnet and testnet)
 // xpubkey: prefix for xpubkeys (we will use 'xpub' for both mainnet and testnet)
 // xprivkey: prefix for xprivkeys (we will use 'xprv' for both mainnet and testnet)
 // networkMagic: used to send messages through the network (not used by us but it's important to set for bitcore-lib, so we use the same as bitcoin)
 // port: used to connect to the network (not used by us but it's important to set for bitcore-lib, so we use the same as bitcoin)
 // dnsSeed: list of dns to connect (not used by us but it's important to set for bitcore-lib, so we use the same as bitcoin)
 const mainnet = Networks.add({
-  name: 'mainnet',
+  name: 'htr-mainnet',
   alias: 'production',
   pubkeyhash: versionBytes['mainnet']['p2pkh'],
   privatekey: 0x80,
   scripthash: versionBytes['mainnet']['p2sh'],
+  bech32prefix: 'ht',
   xpubkey: 0x0488b21e,
-  xprivkey: 0x0488ade4,
+  xprivkey: 0x0488ade5,
```

Comment:
I just think we should have a more detailed explanation of this parameter and `xpubkey` in the code. To prevent any confusion in the future